src/commands/list.rs

```diff
@@ -10,6 +10,10 @@ pub fn list_files(args: &[String])
     let image_fn = args[0].clone();
     let image = fat::Image::from(image_fn)?;
 
+    let params = image.bios_parameter();
+    // Just dump for testing. Remove when tried of me :-)
+    println!("{:?}", params);
```

Comment:
Maybe hide behind a --verbose flag? [Clap](https://github.com/kbknapp/clap-rs) is a pretty excellent command-line parser. Or only compile in in debug builds.